var fail = fun (){
  try{

  }catch(e){

  }
};

fail();
